util/mock-server.js

```diff
@@ -42,6 +47,12 @@ app.post('/mock', (req, res) => {
   res.sendStatus(204);
 });
 
+// Mock an endpoint with a pattern
+app.post('/mock/permanent-pattern', (req, res) => {
```

Comment:
I've implemented stubbing patterns in our mock server. 

An improvement would be to move away from `idapiMockNext`, where it's very easy to stub something unintended.